awssqs/pkg/reconciler/awssqssource.go

```diff
@@ -215,50 +142,13 @@ func (r *reconciler) getReceiveAdapter(ctx context.Context, src *v1alpha1.AwsSqs
 	return nil, apierrors.NewNotFound(schema.GroupResource{}, "")
 }
 
-func (r *reconciler) reconcileEventTypes(ctx context.Context, src *v1alpha1.AwsSqsSource) error {
-	args := r.newEventTypeReconcilerArgs(src)
-	return r.eventTypeReconciler.Reconcile(ctx, src, args)
-}
-
-func (r *reconciler) newEventTypeReconcilerArgs(src *v1alpha1.AwsSqsSource) *eventtype.ReconcilerArgs {
-	spec := eventingv1alpha1.EventTypeSpec{
-		Type:   v1alpha1.AwsSqsSourceEventType,
-		Source: src.Spec.QueueURL,
-		Broker: src.Spec.Sink.Name,
-	}
-	specs := make([]eventingv1alpha1.EventTypeSpec, 0, 1)
-	specs = append(specs, spec)
-	return &eventtype.ReconcilerArgs{
-		Specs:     specs,
-		Namespace: src.Namespace,
-		Labels:    getLabels(src),
-		Kind:      src.Spec.Sink.Kind,
-	}
-}
-
-func (r *reconciler) getLabelSelector(src *v1alpha1.AwsSqsSource) labels.Selector {
+func (r *Reconciler) getLabelSelector(src *v1alpha1.AwsSqsSource) labels.Selector {
 	return labels.SelectorFromSet(getLabels(src))
 }
 
-func (r *reconciler) addFinalizer(s *v1alpha1.AwsSqsSource) {
-	finalizers := sets.NewString(s.Finalizers...)
-	finalizers.Insert(finalizerName)
-	s.Finalizers = finalizers.List()
-}
-
-func (r *reconciler) removeFinalizer(s *v1alpha1.AwsSqsSource) {
-	finalizers := sets.NewString(s.Finalizers...)
-	finalizers.Delete(finalizerName)
-	s.Finalizers = finalizers.List()
-}
-
 func getLabels(src *v1alpha1.AwsSqsSource) map[string]string {
 	return map[string]string{
-		"knative-eventing-source":      controllerAgentName,
+		"knative-eventing-source":      "awssqssource.sources.eventing.knative.dev",
```

Comment:
in other sources we use `eventing.knative.dev/source: controllerAgentName` -- what's the right approach here?
and if it's your implementation here, do we want to use `sources.eventing.knative.dev` ? or just `sources.eventing.dev` ?

Reply:
ah, I'm also realizing that we haven't migrated awssqs to `sources.knative.dev` submitted (will rebase on these changes as needed) #956

Reply:
Yeah I did not change this in this migration.

Reply:
makes sense, but if we're going to change the approach of `controllerAgentName` to `awssqssource.sources.eventing.knative.dev` should we not keep the overall label map consistent (wondering which approach is right, the apply that one moving forward)

Reply:
This is the scheme used in genreconciler